### PR TITLE
Refactor tox.ini for isort

### PR DESCRIPTION
**Pull request by wo0dyn**

### CODE REVIEW

**Comment by brunobord** [comment]
still working on it?

**Comment by wo0dyn** [comment]
> still working on it?

Yes, I need to clean things up here. 
Except if this PR is blocking your work, I'll get back at this once I finish my current work.

**Comment by brunobord** [comment]
this PR https://github.com/novafloss/django-formidable/pull/181 overlaps with some of your work on the "check" part. I guess you'll have to rework on the tox.ini file when it'll be merged.

**Comment by wo0dyn** [comment]
I can squash all my commits into a single one. Let me know.

**Comment by wo0dyn** [comment]
Output from `flake8` and `isort-check`:

```
flake8 create: /home/ubuntu/django-formidable/.tox/flake8
flake8 installdeps: flake8
flake8 installed: appdirs==1.4.0,configparser==3.5.0,enum34==1.1.6,flake8==3.3.0,mccabe==0.6.1,packaging==16.8,pycodestyle==2.3.1,pyflakes==1.5.0,pyparsing==2.1.10,six==1.10.0
flake8 runtests: PYTHONHASHSEED='4244388284'
flake8 runtests: commands[0] | flake8 --version
3.3.0 (mccabe: 0.6.1, pycodestyle: 2.3.1, pyflakes: 1.5.0) CPython 2.7.10 on Linux
flake8 runtests: commands[1] | flake8 --exclude=.git,__pycache__,.tox,.crowdin
isort-check create: /home/ubuntu/django-formidable/.tox/isort-check
isort-check installdeps: isort
isort-check installed: appdirs==1.4.0,isort==4.2.5,packaging==16.8,pyparsing==2.1.10,six==1.10.0
isort-check runtests: PYTHONHASHSEED='4244388284'
isort-check runtests: commands[0] | isort --version

/#######################################################################\

     `sMMy`
     .yyyy-                                                      `
    ##soos##                                                    ./o.
          `     ``..-..`         ``...`.``         `   ```` ``-ssso```
     .s:-y-   .+osssssso/.     ./ossss+:so+:`    :+o-`/osso:+sssssssso/
     .s::y-   osss+.``.``     -ssss+-.`-ossso`   ssssso/::..::+ssss:::.
     .s::y-   /ssss+//:-.`   `ssss+     `ssss+   sssso`       :ssss`
     .s::y-   `-/+oossssso/  `ssss/      sssso   ssss/        :ssss`
     .y-/y-       ````:ssss`  ossso.    :ssss:   ssss/        :ssss.
     `/so:`    `-//::/osss+   `+ssss+-/ossso:    /sso-        `osssso/.
       \/      `-/oooo++/-      .:/++:/++/-`      ..           `://++/.


         isort your Python imports for you so you don't have to

                            VERSION 4.2.5

\########################################################################/

isort-check runtests: commands[1] | cat setup.cfg
[isort]

known_django = django
known_first_party = formidable
sections=FUTURE,STDLIB,DJANGO,THIRDPARTY,FIRSTPARTY,LOCALFOLDER

combine_as_imports = true
; 5 = Hanging Grid Grouped
multi_line_output = 5
not_skip = __init__.py
skip = migrations, settings
line_length = 80
isort-check runtests: commands[2] | isort --check-only --diff --recursive formidable
```

**Comment by wo0dyn** [comment]
µ-benchmark:

| Branch          | Time                                          |
|:---------------:|----------------------------------------------:|
| `master`        | 13.79s user 2.60s system 94% cpu 17.334 total |
| `improve-isort` |  3.87s user 0.25s system 134% cpu 3.068 total | 

```console
$ git checkout master
$ time tox -e flake8,check-python-imports
# ...
tox -e flake8,check-python-imports  13.79s user 2.60s system 94% cpu 17.334 total
$ git checkout improve-isort
$ time tox -e flake8,isort-check
# ...
tox -e flake8,isort-check  3.87s user 0.25s system 134% cpu 3.068 total
```

File: tox.ini [hunk 1]
```diff
@@ -18,28 +20,40 @@ deps =
...
 commands =
+    isort --version
     cat setup.cfg
-    isort --check-only --recursive --verbose formidable
+    isort --check-only --diff --recursive formidable
+
+[testenv:isort-run]
+usedevelop = False
+skip_install = True
+deps =
```
Comment (mgu) [inline tox.ini]: is changedir missing ?

File: tox.ini [hunk 2]
```diff
@@ -18,28 +20,40 @@ deps =
     -rdemo/requirements-demo.pip
 commands =
     python --version
-    pip freeze --local
+    pip list --format=columns
     python manage.py test {posargs}
 
 [testenv:flake8]
+usedevelop = False
+skip_install = True
```
Comment (brunobord) [inline tox.ini]: skip_install! why did I fail to see it?

**Comment by moumoutte** [comment]
> and use option diff iinstead of verbose

Awesome \o/

File: tox.ini (hunk 1 above)
Comment (wo0dyn) [inline tox.ini]: Added in a4a5ed5a8a88aa6269840f871278d3fc571ffc6d. Nice spot.

**Pull request merged**